### PR TITLE
Fixed panels

### PR DESCRIPTION
**Pull request by pistej**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes/no
| New feature?  | yes/no
| Breaks BC?    | yes/no
| Tests pass?   | yes/no
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: src/Module.php [hunk 1]
```diff
@@ -293,10 +293,10 @@ protected function resetGlobalSettings()
      * Gets toolbar HTML
      * @since 2.0.7
      */
-    public function getToolbarHtml()
+    public function getToolbarHtml($tag)
```
Comment (samdark) [inline src/Module.php]: Why?

File: src/models/search/Debug.php [hunk 2]
```diff
@@ -92,20 +93,20 @@ public function attributeLabels()
      */
     public function search($params, $models)
     {
-        $dataProvider = new ArrayDataProvider([
+        $dataProvider = Yii::createObject([
```
Comment (samdark) [inline src/models/search/Debug.php]: Why?

File: src/Module.php (hunk 1 above)
Comment (pistej) [reply src/Module.php]: Few week ago it was only working solution I foud, but now working again. I will revert it.

File: src/models/search/Debug.php (hunk 2 above)
Comment (pistej) [reply src/models/search/Debug.php]: Via constructor can be now set only modue id (not configurable array) and configure new `ArrayDataProvider` can be done via `Yii::createObject` or ArrayDataProvider'setters.

Comment (samdark) [reply src/models/search/Debug.php]: Do we need it to be configurable?

Comment (pistej) [reply src/models/search/Debug.php]: At least you need set data for provider and sorting attributes. In other models is used the same approach: `Yii::createObject`

Comment (samdark) [reply src/models/search/Debug.php]: OK.

**Comment by samdark** [comment]
@pistej is it ready?

**Comment by pistej** [comment]
Yes

**Comment by pistej** [comment]
One question, shouldn't be package "ezyang/htmlpurifier": "~4.6" added into composer required section? Is used in few grids; it was removed from core (I think) and installation is now optionally, but here is used !

**Comment by samdark** [comment]
Let's see how it goes without it. Will add it if there will be problems.

**Comment by samdark** [comment]
Thank you!

**Pull request merged**